Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2568,6 +2603,7 @@ def gen2018HiMix(fragment,howMuch):
                      '--filein':'file:PromptCalibProdEcalPedestals.root'}
 
 steps['RECOHISt4']=steps['RECOHI2015']
+steps['RECOHI2021MIX']=merge([steps['RECOHI2021PPRECO'],{'--pileup':'HiMix','--pileup_input':'das:/RelValHydjetQ_B12_5020GeV_2021/%s/GEN-SIM'%(baseDataSetRelease[9])}])
```

Comment:
@mandrenguyen you may need to introduce the new baseDataSetRelease for 2021 string

Reply:
@mandrenguyen your current setup will call baseDataSetRelease[9] string which is from Run2. I was asking you to implement new index of baseDataSetRelease but yes you're right we never produced any Run3 mixing yet and it will fail. You can continue with what you have for now.